### DISCUSSION TITLE
How big can the lha file be...?

### DISCUSSION THREAD

**Discussion by CommodoreOS** [BlitterStudio/amiberry#930] [Q&A] [answered]
I wanted to know how big lha files can be, I want to make a setup for Workbench but would like to fit Classic Workbench in it, I am asking this cause the A500 Mini uses Amiberry to play WHDLoad games from USB and uses WHDBooster.

**Comment by midwan** [comment, marked as answer]
I think as big as the filesystem allows it - but it will get slower and slower the bigger it gets.
